flink-clients/src/main/java/org/apache/flink/client/ClientUtils.java

```diff
@@ -95,15 +100,55 @@ public static ClassLoader buildUserCodeClassLoader(List<URL> jars, List<URL> cla
 		return FlinkUserCodeClassLoaders.parentFirst(urls, parent);
 	}
 
+	public static JobExecutionResult submitJob(
+			ClusterClient<?> client,
+			JobGraph jobGraph) throws ProgramInvocationException {
+		try {
```

Comment:
You can add the necessary precondition checks like `checkNotNull()` for the `client` and the `jobGraph`.